src/deluge/model/note/note_row.cpp

```diff
@@ -3351,9 +3362,18 @@ Error NoteRow::readFromFile(Deserializer& reader, int32_t* minY, InstrumentClip*
 				uint8_t velocity = hexToIntFixedLength(&hexChars[16], 2);
 				uint8_t lift, probability, iterance, fill;
 
-				if (noteHexLength == 26) { // if reading iterance and fill
-					fill = hexToIntFixedLength(&hexChars[24], 2);
-					iterance = hexToIntFixedLength(&hexChars[22], 2);
+				if (noteHexLength == 28) { // if reading custom iterance and fill
+					fill = hexToIntFixedLength(&hexChars[26], 2);
+					iterance = sanitizeIterance(hexToIntFixedLength(&hexChars[22], 4));
+					probability = hexToIntFixedLength(&hexChars[20], 2);
+					lift = hexToIntFixedLength(&hexChars[18], 2);
+					if (lift == 0 || lift > 127) {
+						goto useDefaultLift;
+					}
+				}
+				else if (noteHexLength == 26) { // if nightly firmware 1.3 with no custom iterances
+					fill = hexToIntFixedLength(&hexChars[26], 2);
```

Comment:
```suggestion
					fill = hexToIntFixedLength(&hexChars[24], 2);
```

I think this is supposed to be 24

Reply:
iterance now has 2 bytes, so that's why the code is different